frontend/actions/Items.js

```diff
@@ -93,3 +93,27 @@ export const getItemVariation = async (name) => fetch(
     }
     return json.payload;
   });
+
+export const updateItemStock = async (item) => fetch(
+    config.baseUrl + config.apis.updateStock, {
```

Comment:
Change this line to utilize the util wrapper created on develop. Every frontend/action function on develop has been wrapped with method as it fixes a bug with production.